Readme Updated (#2)
ArasLabs/generic-action-item: README.md

Release | Notes
--------|--------

> [v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | Readme usage updated

Reviewer: ```suggestion
[v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | Readme usage updated. Tested on 11 SP12, 11 SP15.
```

[v2](https://github.com/ArasLabs/generic-action-item/releases/tag/v2) | Generic Action Item Solution package and Documentation for Aras 10 + 11
[v1](https://github.com/ArasLabs/generic-action-item/releases/tag/v1) | Generic Action Item Solution package and Documentation

#### Supported Aras Versions

Project | Aras
--------|------

> [v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | Tested on 11 SP12, 15

Reviewer: ```suggestion
[v2.0.1](https://github.com/ArasLabs/generic-action-item/releases/tag/v2.0.1) | 10 SPx, 11 SPx
```

8. Click **Import** in the top left corner.
9. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\2-Import - Example on DCO (optional)\1-Import\imports (admin).mf`.
10. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\2-Import - Example on DCO (optional)\2-Import\imports (admin).mf`.
11. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\3-Import - Example Action Item Template (optional)\imports (admin).mf`.
12. Close the Aras Package Import tool.

## Usage
This package can be used one of two ways:

> * by use of an Action Item Template set up by an Administrator

Reviewer: ```suggestion
* Use an Action Item Template set up by an Administrator
```

> * Added to an existing DCO item by a user either as ad-hoc or by using an existing template as above

Reviewer: ```suggestion
* Add to an existing DCO item by a user, either as ad-hoc or by using an existing template as above.
```

> This item may also be configured as a relationship to other items, but must be set to a Create New type of relationship.

###Example Usage: 

> 1. (Optional) Create Template(s) for action items

Reviewer: ```suggestion
1. (Optional) Create Template(s) for action items.
```

> 2. Create New or Open Existing Express DCO

Reviewer: ```suggestion
2. Create or open an Express DCO.
```

>   * if new, must save item before using template action

Reviewer: ```suggestion
  * If the item is new, save item before using template action.
```

> 3. Use Action to add from template or add ad hoc using tab

Reviewer: Try to be very specific about how people need to do things. Which action? How do they run that action? Is it a toolbar button, main menu action, or context menu action?

> 3. Use Action to add from template or add ad hoc using tab

Author: ```suggestion
3. Use Actions > Add Action Item From Template to add from template or create a new relationship to an ad hoc item using the Action Item tab (Both methods can be used together)
```

Reviewer: 👍

> 4. Fire workflow for action using RMB action in search grid/tab

Author: ```suggestion
4. Start Workflow for action using RMB action in the Action Item admin search grid or the Action Item tab on a related item
```

Reviewer: 👍

5. Complete action items via inbasket (closing item will require user password)

>   * if notes are needed to close, add to action item via RMB view command

Reviewer: ```suggestion
  * If notes are needed to close, add to action item via RMB view command.
```
